pipeline/utilities.py

```diff
@@ -32,9 +39,16 @@ def find_session_matched_nwbfile(sess_data_dir, animal_id, date_of_experiment):
                 continue
             # read subject_id out of this file
             subject_id = temp_nwb['general']['subject']['subject_id'].value.decode('UTF-8')
-            # -- session_time 
-            session_start_time = temp_nwb['session_start_time'].value
-            session_start_time = parse_prefix(session_start_time)
+            
+            # -- session_time - due to error in extracellular dataset (session_start_time error), need to hard code here...
+            if which_data == 'whole_cell':  # case: whole cell
+                session_start_time = temp_nwb['session_start_time'].value
+                session_start_time = parse_prefix(session_start_time)
```

Comment:
take `session_start_time = ...` out of the `if .. then ..`  if it's the same for all cases.